addons/sourcemod/scripting/RepeatKillDetector.sp

```diff
@@ -85,10 +87,24 @@ public Action Command_ForceRepeator(int client, int argc)
 	}
 
 	CShowActivity2(client, "{green}[ZR] {olive}", "{green}%s{default} the Repeat killer protection. %s respawn for this round.", (value ? "Enabled" : "Disabled"), (value ? "Disabled" : "Enabled"));
-	LogAction(client, -1, "[ZR] %L %s the Repeat killer protection. \n[ZR]%s respawn for this round.", client, (value ? "Enabled" : "Disabled"), (value ? "Disabled" : "Enabled"));
-	
+	LogAction(client, -1, "[ZR] %L %s the Repeat killer protection. \n[ZR] %s respawn for this round.", client, (value ? "Enabled" : "Disabled"), (value ? "Disabled" : "Enabled"));
+
 	if(bValue)
+	{
 		g_bBlockRespawn = false;
+		
+		if(!g_bZombieSpawned)
+			return Plugin_Handled;
+			
+		for (int i = 1; i < MaxClients; i++)
+		{
+			if(!IsClientInGame(i))
+				continue;
+				
+			if(!IsPlayerAlive(i) && GetClientTeam(i) > 1)
+				FakeClientCommand(i, "say /zspawn");
```

Comment:
avoid using this function since some commands dont work with it

Instead expose ZSpawnClient here in the api:
- https://github.com/srcdslab/sm-plugin-zombiereloaded/blob/master/src/addons/sourcemod/scripting/zr/zspawn.inc#L306

Or use ZR_RespawnClient:
- https://github.com/srcdslab/sm-plugin-zombiereloaded/blob/master/src/addons/sourcemod/scripting/include/zr/respawn.zr.inc#L47